src/webui/api/rsscontroller.h

```diff
@@ -44,6 +44,7 @@ private slots:
     void removeItemAction();
     void moveItemAction();
     void itemsAction();
+    void refreshItem();
```

Comment:
shouldn't it have "Action" at the end of name?

Reply:
Oops... Yes, of course. Thank you!

Reply:
Fixed.